src/models/constants.ts

```
@@ -196,6 +196,7 @@ export const defaultData: IPersonalData2 = {
     schemaVersion: 2,
     modifiedDate: undefined,
     seenAppVersion: undefined,
+    playerMetadata: { playerName: undefined, powerLevel: undefined },
```

Biome (2.4.15)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)